README.md

```diff
@@ -178,6 +178,71 @@ excludes all algorithms of the "Sphincs" family.
 *Note*: By default, interoperability testing with oqs-openssl111 is no longer
 performed by default but can be manually enabled in the script `scripts/runtests.sh`.
 
+Building under Windows
+--------------------
+The instructions below were tested on Windows 10 and 11 using MSYS2 MINGW64 (building with Visual Studio 2019 was NOT successful in our case). They are a bit experimental and only provisional. Note that the tests were skipped in the building process, however, setting up a test server and client with PQC algorithms still works.
+
+## Pre-requisites
+
+Open MSYS2 MINGW64 and make sure that git is installed:
+
+    pacman -S git
+
+Create a new directory `.local`:
+
+    mkdir install
+
+## OpenSSL 3
+
+Build and install OpenSSL 3 in `.local`:
+
+    git clone git://git.openssl.org/openssl.git
+    cd openssl
+    ./config --prefix=$(echo $(pwd)/../.local) && make && make install_sw
+    cd ..
+
+For [OpenSSL implementation limitations, e.g., regarding provider feature usage and support,
+see here](https://wiki.openssl.org/index.php/OpenSSL_3.0#STATUS_of_current_development).
+
+
+## liboqs
+
+Build and install `liboqs` in `.local`:
+
+    git clone https://github.com/open-quantum-safe/liboqs.git
+    cd liboqs
+    cmake -DCMAKE_INSTALL_PREFIX=$(pwd)/../.local -S . -B _build
+    cmake --build _build && cmake --install _build
+    cd ..
+
+Further `liboqs` build options are [documented here](https://github.com/open-quantum-safe/liboqs/wiki/Customizing-liboqs).
+
+## Building the provider
+
+Create a file `oqsprov.def` in the directory `oqsprov` with the following lines:
+
+    EXPORTS
+        OSSL_provider_init
+
+Add `oqsprov.def` to the following section in CMakeLists.txt of the directory `oqsprov` in line 8.
+
+    set(PROVIDER_SOURCE_FILES
+        oqsprov.c oqsprov_capabilities.c oqsprov_keys.c
+        oqs_kmgmt.c oqs_sig.c oqs_kem.c
+        oqs_encode_key2any.c oqs_endecoder_common.c oqs_decode_der2key.c oqsprov_bio.c
+        oqsprov.def
+    )
+
+Modify `tests/test_common.c` in line 27 by replacing `while((comma = strchr(alglist, ','))) {` with :
```

Comment:
This is highly risky (line numbers change with code changes). Better would be a permalink, such as https://github.com/open-quantum-safe/oqs-provider/blob/5dc7df1506256f43a31b01d97110c10648f1a4d2/test/test_common.c#L27

That however makes me wonder: Which code base are you referring-to in this README change? It seems the code already does use `index` instead of `strchr`, thus making this modification suggestion moot, no? Or is the suggestion to use `strchr` instead of `index`?